src/main/java/sirius/db/mixing/query/constraints/FilterFactory.java

```diff
@@ -334,6 +334,14 @@ public C lteOrEmpty(Mapping field, @Nullable Object value) {
      */
     public abstract C notFilled(Mapping field);
 
+    /**
+     * Generates a constraint which ensures that the given field exists.
+     *
+     * @param field the field to check
+     * @return the generated constraint
+     */
+    public abstract C exists(Mapping field);
```

Comment:
shouldn't we just implement this in the factories which actually support this method?